region/lowernorfair/east.json

```diff
@@ -1745,41 +1822,55 @@
                   ]},
                   {"and": [
                     "Bombs",
-                    {"cost": [
-                      {"heatFrames": 50}
-                    ]}
+                    {"cost": [{"heatFrames": 50}]}
                   ]}
-                ]},
+                ]},                
                 {"or": [
                   {"cost": [
-                    {"heatFrames": 550},
+                    {"heatFrames": 50},
                     {"enemyDamage": [{
                       "enemy": "Dessgeega",
                       "type": "contact",
                       "hits": 1
                     }]}
                   ]},
-                  {"and": [
-                    "ScrewAttack",
-                    {"cost": [
-                      {"heatFrames": 650}
-                    ]}
-                  ]},
-                  {"and": [
-                    {"cost": [
-                      {"heatFrames": 950}
-                    ]},
-                    {"or": [
-                      "Super",
+                  {"obstacle":{
+                    "id": "C",
+                    "requires": [
+                      {"obstacle":{
+                        "id": "D",
+                        "requires": null
+                      }},
                       {"and": [
```

Comment:
Since we're at the first level of the obstacle's `requires`, this `and` is implicit.

But I think it's probably meant to be part of a big `or` that's missing?

Reply:
Indeed I'm missing an "or" even if this Obstacle D won't be part of it. Ops!

Reply:
I imagine they would not be, indeed. I also imagine an obstacle within an `and` can only be considered destroyed if the `and` is actually used to traverse a link.